com.unity.render-pipelines.high-definition/Runtime/Material/Lit/Lit.hlsl

```diff
@@ -207,6 +207,12 @@ float GetAmbientOcclusionForMicroShadowing(BSDFData bsdfData)
     return sourceAO;
 }
 
+#define HAS_PAYLOAD_WITH_UNINIT_GI
+float GetUninitializedGIPayload(SurfaceData surfaceData)
```

Comment:
I feel bad about this one, we will need to add it to all our lit material (hair, fabric, eye, staacklit...). But guess we have no choice. Problem is very similar to 
GetAmbientOcclusionForMicroShadowing().
Let's let it for now for the experimental development, but not sure we have alternative. At least we should change the name to GetAmbientOcclusionFromSurface() so we can reuse it for other purpose

Reply:
We don't  need to add it to everything that's why I added the define, we only need to add it to deferred materials.

Reply:
The reason why I avoided using AmbientOcclusion in that name is exactly to allow potential reuse for something else.